benchmarl/experiment/experiment.py

```diff
@@ -1035,7 +1038,13 @@ def reload_from_file(restore_file: str) -> Experiment:
             critic_model_config = pickle.load(f)
             callbacks = pickle.load(f)
         task.config = task_config
+        experiment_config.save_folder = experiment_folder.parent
```

Comment:
This was like this on purpose, I wanted to leave the freedom to restart in a new folder, maybe we can have an input (default false) that decides weather to reload in the same folder